lib/nerves_hub_cli/api.ex

```diff
@@ -157,11 +162,10 @@ defmodule NervesHubCLI.API do
       is_atom(ca_store) and !is_nil(ca_store) ->
         ca_store.ca_certs()
 
-      Code.ensure_loaded?(CAStore) ->
-        CAStore.file_path()
-        |> File.read!()
-        |> X509.from_pem()
-        |> Enum.map(&X509.Certificate.to_der/1)
+      # TODO: These conditions can probably be removed after successfully testing for some time
```

Comment:
TODO: check security considerations of loading the CA certs at compile time into the binary, and the checks here